test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -412,6 +412,15 @@ def test_purchase_and_void
     assert_successful_response(void)
   end
 
+  def test_void_with_no_authorization_value
+    merchant_transaction_id = 'testTransaction131' + SecureRandom.hex(3)
+
+    @gateway.authorize(@amount, @credit_card, @options.merge(merchant_transaction_id:))
```

Comment:
It looks like you are setting the key in the merge, but not giving it a value `.merge(merchant_transaction_id:))`

Reply:
Ah, thanks for the explantation!